powertools-serialization/pom.xml

```diff
@@ -45,6 +45,14 @@
             <groupId>io.burt</groupId>
             <artifactId>jmespath-jackson</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-lambda-java-events</artifactId>
```

Comment:
In v4 of the events lib, there will be a dependency on Jackson, I don't think that changes anything, just worth noting.